CIP-Fair_Stakepool_Rewards/README.md

```diff
@@ -0,0 +1,195 @@
+---
+CIP: 75
+Title: Fair Stake Pool Rewards
+Status: Proposed
+Category: Reward-Sharing Scheme
+Authors:
+    - Tobias Fancee <tobiasfancee@gmail.com>
+Implementors:
+    - Input Output Global
+    - Cardano Foundation
+    - Emurgo
+Discussions:
+    - https://forum.cardano.org/t/cip-fair-stakepool-rewards/109368
```

Comment:
In my opinion the discussion on the Cardano Forum is fragmented and should also include, if not this link itself (and maybe somewhere except the preamble), then some of the ideas mentioned in this separate discussion thread: https://forum.cardano.org/t/cip-improved-rewards-scheme-parameters/112409